styles.css

```diff
@@ -522,15 +522,18 @@ footer .ending-line {
   }
 
   .header .welcome {
+    padding: 25px 25px auto auto;
     margin-left: 70px;
   }
 
   .header .job-desc {
     margin-left: 70px;
+    padding: 25px 25px auto auto;
   }
 
   .header .intro {
     margin-left: 70px;
+    padding: 25px 25px auto auto;
   }
 
```

Comment:
- Hey, great adding the adjustment as pointed by the previous reviewer, however, the padding issue is not yet resolved as there's no padding bottom in the `header` as this is making your design not consistent enough. see the images below to see the highlight to implement:

*Your design*
![Screenshot from 2023-02-23 17-18-45](https://user-images.githubusercontent.com/59397159/220968410-e9cadebf-5a27-4d4b-9d82-aac0ad9d0202.png)

*Figma template*
![Screenshot from 2023-02-23 17-26-10](https://user-images.githubusercontent.com/59397159/220968749-19eafd89-3417-416b-b305-c9b61d14c1c5.png)